docs/manual-EN/2.query-language/0.README.md

```diff
@@ -1,3 +1,66 @@
 # Reader
 
 This chapter is for those who want to use `Nebula Graph` query language.
+
+## Placeholder Identifiers and Values
+
+The query language of **Nebula Graph**, i.e. the nGQL, refers to the following standards in design:
+
+- ISO/IEC 10646
+- ISO/IEC 39075
+- ISO/IEC NP 39075 (Draft)
+
+In template code, any token that is not a keyword, a literal value, or punctuation is a placeholder identifier or a placeholder value.
+
+Tokens details please refer to the following table:
+
+|  Token | Meaning  |
+|  ----  | ----  |
+| < >    | name of a syntactic element |
+| ::=    | formula that defines an element |
+| [ ]    | optional elements |
+| { }    | explicitly specified elements |
+|  \|    | complete alternative elements |
+| ...    |  may be repeated any number of times |
+
+Example:
+
+```bash
+INSERT VERTEX <tag_name> [, <tag_name>, ...] ( <prop_name_list>[, <prop_name_list> ...])
+     {VALUES | VALUE} <vid>: ( <prop_value_list> [, <prop_value_list> ...])
+
+prop_name_list:
+  [prop_name [, prop_name] ...]
+
+prop_value_list:
+  [prop_value [, prop_value] ...]
+```
+
+The user-defined identifiers are tag_name, prop_name_list and prop_value_list.
+
+- Choices
+
+    The vertical bar `|` is used to separate the choices, when the syntax requires that the user choose one out of a set of values. Example:  Either the keyword TAG or EDGE is to be used.
+
+    Template:
+
+    ```bash
+    ALTER TAG | EDGE <tag_name> | <edge_name>
+    <alter_definition> [, alter_definition] ...]
+    [ttl_definition [, ttl_definition] ... ]
+    ```
+
+- Optional content
+
+    Square brackets are used to enclose a portion that is optional.  Options can be nested. Square brackets themselves are rarely used as part of the nGQL language itself.
+
+    Example: In the CREATE SPACE statement, partition_num is optional. If used, partition_num is to be followed by a value.
```

Comment:
partition_num is a field or property, which should be enclosed by `` according to the styles of doc